server/services/facebookStrategy.js

```diff
@@ -0,0 +1,29 @@
+import passport from 'passport';
+import {
+  Strategy as FacebookStrategy
+} from 'passport-facebook';
+import dotenv from 'dotenv';
+
+import generateToken from '../utilities/jwtGenerator';
+import passportAuthenticator from '../utilities/passportAuthentication';
+import userRepo from '../repository/userRepository';
+
+dotenv.config();
+
+passport.use(new FacebookStrategy({
```

Comment:
Can we have the social configs in the config directory? Create them there then import into this file

Reply:
Ok will do this